backend/data/src/main/kotlin/io/tolgee/service/notification/NotificationService.kt

```diff
@@ -19,5 +23,19 @@ class NotificationService(
 
   fun save(notification: Notification) {
     notificationRepository.save(notification)
+    websocketEventPublisher(
+      "/users/${notification.user.id}/${WebsocketEventType.NOTIFICATIONS_CHANGED.typeName}",
+      WebsocketEvent(
+        actor = ActorInfo(
```

Comment:
@JanCizmar I am not sure about this field, but it was not-nullable.

Should it be the user who created the task, in this case?

Reply:
You can make it nullable.